### PR TITLE
docs: chinese translation for zh development & what's new

### PR DESCRIPTION
**Pull request by zpi1223**
Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Chinese translation of zh development modules

### CODE REVIEW

File: docs/zh/development/tests.md [hunk 1]
```diff
@@ -1 +0,0 @@
-../../en/development/tests.md
```
Comment (tanghongtanghong) [inline docs/zh/development/tests.md:1]: 得到的

Comment (tanghongtanghong) [reply docs/zh/development/tests.md:1]: ../../en/development/tests.md

Comment (zpi1223) [reply docs/zh/development/tests.md:1]: I don’t understand what you mean, can I express it more clearly?

Comment (zpi1223) [reply docs/zh/development/tests.md:1]: @tanghongtanghong I don’t understand what you mean, can you express it more clearly?

**Comment by zpi1223** [comment]
@kitaisreal I have two questions：
First of all, I'm confused. I don't know what @tanghongtanghong  wants to express. Now all the checks have passed, so I don't know what I should do next. Could you give me some advice?
Secondly, I am currently translating 2020.md of whats new change-log. If merge is possible, could you wait until I finish translating 2020.md?

**Comment by kitaisreal** [comment]
@zpi1223 yes, lets wait until 2020.md will be translated.

**Comment by zpi1223** [comment]
> @zpi1223 yes, lets wait until 2020.md will be translated.

Thank you very much for your prompt reply. I don’t know if I need to pay attention to what @tanghongtanghong  wants to express.

**Comment by zpi1223** [comment]
> @zpi1223 yes, lets wait until 2020.md will be translated.

Hi @kitaisreal , thank you for your waiting these few days. I have translated 2020.md and passed the relevant inspections. When your time is convenient, you can proceed to the next Merge operation. Thank you very much.

**Pull request closed without merging**